src/status_im/ui/screens/chat/views.cljs

```diff
@@ -421,7 +426,8 @@
        :onMomentumScrollEnd          state/stop-scrolling
        ;;TODO https://github.com/facebook/react-native/issues/30034
        :inverted                     (when platform/ios? true)
-       :style                        (when platform/android? {:scaleY -1})})]))
+       :style                        (when platform/android? {:scaleY -1})
+       :on-layout                    on-messages-view-layout})]))
```

Comment:
this implementation only for the new design, so you don't need this event for old one